spec.md

```diff
@@ -1015,7 +1015,7 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 
 A Controller Plugin MUST implement this RPC call if it has `CREATE_DELETE_VOLUME` capability.
 This RPC will be called by the CO to deprovision a volume.
-If successful, the storage space associated with the volume MUST be released and all the data in the volume SHALL NOT be accessible anymore.
+If successful, the storage space associated with the volume MAY or MAY NOT be released depending on the reclaim or retention policy specified on the storage system.
```

Comment:
The only nit I have here is the use of "reclaim policy" that gives the impression that this may be related to Kubernetes reclaim policy, but in fact it should be soley based upon the backend and it's configuration no?

Reply:
@xing-yang  Yeah, maybe omit the "if successful" and then maybe just something like:

> The space associated with teh volume MAY or MAY NOT be released depending upon the configuration of the back end device including any available retention policies.

The only thing for me was "retention policy" being a parameter in Kubernetes it just made it seem too specific.  What do you think?

Reply:
@j-griffith I like it.  I updated the PR with your suggestion.  I still added "If successful, ..." to start with the sentence.